packages/backend/src/studio.ts

```diff
@@ -66,18 +66,34 @@ export class Studio {
     this.#extensionContext = extensionContext;
   }
 
+  private checkVersion(): boolean {
+    if (!version) return false;
+
+    const current = coerce(version);
+    if (!current) return false;
+
+    if (current.major === 0 && current.minor === 0) {
+      console.warn('nighties version are not subject to version verification.');
```

Comment:
Nighties are something women wear to bed, please fix typo to nightlies throughout. ;-)

Reply:
Thanks for raising this !!!